frontend/src/common/queries/collections.ts

```diff
@@ -173,8 +245,8 @@ export function useCollectionUploadLinks(
   const queryCache = useQueryCache();
 
   return useMutation(collectionUploadLinks, {
-    onSuccess: () => {
-      queryCache.invalidateQueries([USE_COLLECTION, id, visibility]);
+    onMutate: () => {
+      // queryCache.invalidateQueries([USE_COLLECTION, id, visibility]);
```

Comment:
these mutations were breaking the collection view, need to take a look at them in a separate issue.  Disabled for now